Code/Tools/ProjectManager/Source/ProjectsScreen.h

```diff
@@ -102,7 +115,9 @@ namespace O3DE::ProjectManager
         AZStd::unordered_map<AZ::IO::Path, ProjectButton*> m_projectButtons;
         QList<ProjectInfo> m_requiresBuild;
         QQueue<ProjectInfo> m_buildQueue;
+        QQueue<ProjectInfo> m_exportQueue;
         ProjectBuilderController* m_currentBuilder = nullptr;
+        ProjectExportController* m_currentExporter = nullptr;
```

Comment:
Maybe we can use an `AZStd::unique_ptr` instead of a raw pointer?

Reply:
I'm noticing alot of the feedback in this PR also applies to the Builder components as well (as that's where I copied most of the code). Should I only make changes for export, or do we need to update builder as well?

I'd like to keep it focused on export for the time being, and make updating builder as a seperate PR/issue if possible

Reply:
Just only on the changes/new code. It shouldve been a smart ptr to begin with, but we want to focus only on export-project changes

Reply:
got it, I'll make the changes then